clear_postgres_tables.sql

```diff
@@ -1,7 +1,6 @@
 /*
```

Comment:
This is a practice from my previous company - scripts that do deletes like this are commented out so that you don't accidentally delete all your data.